CMakeLists.txt

```diff
@@ -216,8 +216,12 @@ set(MD_VERSION ${MD_VERSION})
 configure_file(libmd.pc.in ${CMAKE_BINARY_DIR}/libmd.pc @ONLY)
 install (FILES ${CMAKE_BINARY_DIR}/libmd.pc DESTINATION lib/pkgconfig)
 
-install (TARGETS ${PROJECT_NAME} DESTINATION lib)
-install (FILES "${BS_DIR_INC}/common/instrumentation/api/metrics_discovery_api.h" DESTINATION include)
+# Library/header
+if (${LIB_INSTALL_DIR} STREQUAL "")
```

Comment:
Should be 
``if ("${LIB_INSTALL_DIR}" STREQUAL "")`` ("" added around LIB_INSTALL_DIR)

Reply:
:+1: